packages/http-client/src/index.ts

```diff
@@ -1,18 +1,19 @@
 import fetch from "isomorphic-fetch";
 import type { Includes, Project } from "./index.d";
 
-const protocol = process.env.NODE_ENV === "production" ? "https" : "http";
-
 export const loadProject = async ({
+  webstudioAPIUrl = null,
   projectId,
-  host = process.env.DESIGNER_HOST || "localhost:3000",
   include = { tree: true, props: true, breakpoints: true },
 }: {
+  webstudioAPIUrl: string | null;
   projectId: string;
-  host?: string;
   include?: Includes<boolean>;
 }): Promise<Project> => {
-  const baseUrl = new URL("/", `${protocol}://${host}`);
+  if (!webstudioAPIUrl) {
```

Comment:
once you only have undefined
```suggestion
  if (webstudioAPIUrl === undefined) {
```

Reply:
wait this doesn't make any sense, this URL is not optional and so shouldn't accept a null at type level, but then validate in code to not accept null, it should only accept string strictly and so you don't even have to do this runtime check

Reply:
Unless you think the url could come from env and be undefined or empty string, in this case you could do new URL(apiUrl) and then check for hostname for example, so you have an actual validation